pkg/webhooks/admission/queues/validate/validate_queue.go

```diff
@@ -255,10 +257,21 @@ func validateHierarchicalQueue(queue *schedulingv1beta1.Queue) error {
 		return fmt.Errorf("failed to get parent queue of queue %s: %v", queue.Name, err)
 	}
 
-	if parentQueue.Status.Pending+parentQueue.Status.Running+parentQueue.Status.Unknown+parentQueue.Status.Inqueue > 0 {
-		return fmt.Errorf("queue %s cannot be the parent queue of queue %s because it has PodGroups (pending: %d, running: %d, unknown: %d, inqueue: %d)",
-			parentQueue.Name, queue.Name, parentQueue.Status.Pending,
-			parentQueue.Status.Running, parentQueue.Status.Unknown, parentQueue.Status.Inqueue)
+	pgList, err := config.VolcanoClient.SchedulingV1beta1().PodGroups("").List(context.Background(), metav1.ListOptions{})
```

Comment:
How about use `informer` list

Reply:
Informer will list/watch all podgroups, which increases webhook memory consumption. I don’t really want to sacrifice memory for a verification.

Reply:
But not using the informer will put more pressure on the apiserver.

Reply:
OK, I'll solve it

Reply:
cc @Monokaix

Reply:
We can check queue.status.allocated.